apps/src/p5lab/P5LabView.jsx

```diff
@@ -188,7 +188,7 @@ class P5LabView extends React.Component {
           />
         )}
         <VisualizationResizeBar />
-        <InstructionsWithWorkspace>
+        <InstructionsWithWorkspace labType={this.props.labType.toLowerCase()}>
```

Comment:
P5LabView is shared between Game Lab and Sprite Lab (including Sprite Lab variants like Poetry)